app/routes/git_tokens.py

```
@@ -1,204 +1,343 @@
+            user_id=payload.user_id,
+            git_hosting=payload.git_hosting,
+            token_value=encrypted_token,
+            username=user.get("username", ""),
+        )
+
+        return APIResponse.success(
+            message=constants.TOKEN_SAVED_SUCCESSFULLY, data={"id": str(git_label.id)}
+        )
+    except Exception:
+        return APIResponse.error(
+            message=f"Failed to save GitLab token: {str(e)}",
```

Ruff (0.11.9)

Undefined name `e`

(F821)